misc/discoverynode/README.md

```diff
@@ -1,8 +1,121 @@
-# Discovery
+# Discovery Node
 
-## UDMI MQTT Client
+## Notes
 
-`sudo ~/venv/bin/python3 main.py`
+`vendor` discovery family is a counter which increments every second
```

Comment:
This doesn't make sense?

Reply:
It's just a note to say that if the discovery node does a `vendor` discovery is actually a counter

```python
 for i in itertools.count(1):
      if self.cancelled:
        return
      result = DiscoveryEvent(
          generation=self.generation, scan_family=self.scan_family, scan_addr=i
      )
      self.publisher(result)
      time.sleep(1)
```